js_modules/dagit/packages/core/src/runs/RunListTabs.new.tsx

```diff
@@ -0,0 +1,135 @@
+import {gql} from '@apollo/client';
+import {Colors, JoinedButtons, TokenizingFieldValue} from '@dagster-io/ui';
+import isEqual from 'lodash/isEqual';
+import * as React from 'react';
+import {useLocation} from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import {RunStatus} from '../graphql/types';
+import {useDocumentTitle} from '../hooks/useDocumentTitle';
+import {AnchorButton} from '../ui/AnchorButton';
+
+import {doneStatuses, inProgressStatuses, queuedStatuses} from './RunStatuses';
+import {runsPathWithFilters, useQueryPersistedRunFilters} from './RunsFilterInput';
+
+const getDocumentTitle = (selected: ReturnType<typeof useSelectedRunsTab>) => {
+  switch (selected) {
+    case 'all':
+      return 'Runs | All runs';
+    case 'done':
+      return 'Runs | Done';
+    case 'in-progress':
+      return 'Runs | In progress';
+    case 'queued':
+      return 'Runs | Queued';
+    case 'scheduled':
+      return 'Runs | Scheduled';
+    default:
+      return 'Runs';
+  }
+};
+
+interface Props {
+  queuedCount: number | null;
+  inProgressCount: number | null;
+}
+
+export const RunListTabs: React.FC<Props> = React.memo(({queuedCount, inProgressCount}) => {
+  const [filterTokens] = useQueryPersistedRunFilters();
+  const selectedTab = useSelectedRunsTab(filterTokens);
+
+  useDocumentTitle(getDocumentTitle(selectedTab));
```

Comment:
Totally unrelated to this PR, but it's a little odd for the tabs component to be the thing that sets the document title. 😅 Guess we've had it that way for a long time!